src/components/NodesTable/NodesTable.vue

```diff
@@ -21,7 +21,16 @@ export default defineComponent({
   },
   setup() {
     const store = useStore()
-    const nodes = computed(() => store.getters['nodes/list'])
+    const nodes = computed(() => {
+      const arr = store.getters['nodes/list']
+      return arr.sort((a, b) => {
+        if (/^http:\/\//.test(a.url) || /^http:\/\//.test(b.url)) {
+          return a.url > b.url ? -1 : b.url > a.url ? 1 : 0;
```

Comment:
```suggestion
          return a.url > b.url ? 1 : b.url > a.url ? -1 : 0;
```

Reply:
Are you sure you sort it correctly?

Reply:
Try sorting array from this order. You will see that the sorting function doesn't work properly.

```js
[
  "https://bid.adamant.im",
  "http://78.47.205.206:36666",
  "http://5.161.53.74:36666",
  "https://clown.adamant.im",
  "http://23.226.231.225:36666",
  "https://debate.adamant.im",
  "https://endless.adamant.im",
  "https://lake.adamant.im",
  "https://unusual.adamant.im"
].sort(yourSortingFn)
```

![image](https://github.com/Adamant-im/adamant-im/assets/25831507/768f6c35-fee6-4457-ba3a-a7b544ef686a)

Reply:
I think it's because in the console you don't have the object itself, a.url and b.url are undefined
<img width="1439" alt="Screenshot 2023-07-26 at 11 11 47 PM" src="https://github.com/Adamant-im/adamant-im/assets/50330577/f2da2ea5-d1fc-4219-908b-843cc831dc98">

Reply:
My mistake